akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -194,22 +194,24 @@ class ActorLoggingSpec extends ActorTestKit with TypedAkkaSpec {
 
     "provide the MDC values in the log" in {
       val behaviors = Behaviors.withMdc[Protocol](
-        { (msg) ⇒
+        Map("static" -> 1),
+        // FIXME why u no infer the type here Scala??
+        { (msg: Protocol) ⇒
```

Comment:
very weird, would be good to know why?

Reply:
Really weird given that `def a[T](whatever: B)(T => C)` and we explicitly call `a[T](b)(c)`.